README.md

```diff
@@ -22,14 +22,18 @@ In case you have installed Trezor Bridge previously there will be a port conflic
 #### Prerequisites
 
 You need:
-- Docker (to run NixOS in a container).
+- Docker (to run NixOS in a container). 
+- xhost
+
+On NixOS, simply enter the `nix-shell`, it is all there.
```

Comment:
Here in master, we have two nix-shells, so we could specify `in the root directory`  - but not necessary, if we will merge it soon with `incoming`, where we have united the nix-shell

Reply:
Fixed by rebasing, please check.

Reply:
Port 9002 is right, and there are no other breaking changes. So we can merge! (When you were the one creating the commit, should it be me to press the Green button?)

Reply:
Yes, please approve the PR and feel free to merge then :).